js/colourpicker.js

```diff
@@ -539,7 +539,6 @@
       input.data('colour',rgb);
       hex = rgb2hex(rgb);
       hsb = rgb2hsb(rgb);
-      var alpha = allowAlpha ? rgb.a || 1 : false;
```

Comment:
this variable was unnecessary and caused a weird bug where the alpha slider didn't reset to the right place.